_includes/tip-of-the-month-intro.html

```diff
@@ -0,0 +1,11 @@
+{% capture tip-of-the-month-intro %}
+Each month we seek to provide a software tip of the month geared towards helping you achieve your software goals. Views
+expressed in the content belong to the content creators and not the organization, its affiliates, or employees. If you
+have any software questions or suggestions for an upcoming tip of the week, please don't hesitate to reach out!
```

Comment:
*"Tip of the week" here?

Maybe we should just call this `tip-intro.html` and make the language more vague to give us some leniency for when we make a post?

Reply:
Thanks for this, great point! I'll make the update here.

I made this change as I noticed the DBMI newsletters had changed to this nomenclature ("tip of the month" vs "tip of the week") and I don't think I can keep up with weekly publishing anymore. The previous "tips" are sometimes many per month, so I tried to keep the date/timing mentions consistent + truthful to when they were published (it might be time consuming to update the publish dates for all). Maybe simplest would be to drop the prefix of "tip of the x" to provide flexibility on when things get published, by whom, and aligning communications materials. I don't have a strong opinions here either way.

Reply:
Thinking this through a little more, and given no strong focus on time / cadence, I think I understand your comment a bit better. I moved to a more generic intro to make these more flexible (for example, tip of the month can still hold and be perspectival to any audience sharing them). Thanks again!